syntax = "proto3";

package fs.common;

// 为了让 brpc 生成 Service/Stub（proto3 下需要）
option cc_generic_services = true;

enum StatusCode {
    OK = 0;
    NOT_FOUND = 1;
    IO_ERROR = 2;
    ALREADY_EXISTS = 3;
}
